app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -0,0 +1,61 @@
+module PublishingApi
+  class MinistersIndexPresenter
+    attr_accessor :item
+    attr_accessor :update_type
+
+    def initialize(item: nil, update_type: nil)
+      self.item = nil
+      self.update_type = update_type || "major"
+    end
+
+    delegate :content_id, to: :item
+
+    def content
+      content = BaseItemPresenter.new(
+        item,
+        title: 'ministers_index',
+        update_type: update_type,
+      ).base_attributes
+
+      content.merge!(
+        base_path: base_path,
+        details: details,
+        document_type: 'ministers_index',
+        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        schema_name: "ministers_index",
+      )
+
+      if I18n.locale.to_s == "cy"
+        content.merge!(
+          PayloadBuilder::Routes.for(base_path, additional_routes: ["cy"])
+        )
+      else
+        content.merge!(
+          PayloadBuilder::Routes.for(base_path)
+        )
+      end
+    end
+
+  private
+
+    def details
+
+      details = {}
+      setting = SitewideSetting.find_by(key: :minister_reshuffle_mode)
+
+      if setting 
+        details = {
+          :reshuffle => { 
+            :message => setting.on ? setting.govspeak : nil
+          }.compact
+        } 
+      end
+      details
```

Comment:
```suggestion
      return {} unless setting.on

      { reshuffle: { message: setting.govspeak } }
```

This simplifies the code quite a bit and I think makes it easier to read.

Reply:
Personally, I think there is some value in having some indication in the Content Item that there's something going on with the reshuffle mode when it's not on.

I know there's the JSON schema, but you might not think to look there if you look at the content item, and don't see anything related to the reshuffle (because it's not toggled on currently).

Reply:
I don't mind either way, I do like the idea of the `reshuffle` field always being present. We should probably [change the schema to make `message` the optional part](https://github.com/alphagov/govuk-content-schemas/blob/327a3ae972381106a401825ab79bece8184fc1c5/formats/ministers_index.jsonnet#L7-L12) if we decide to do this.